redesign/src/styles/layout/_footer.scss

```diff
@@ -0,0 +1,93 @@
+footer {
+    height: 380px;
+    width: auto;
+    z-index: 3;
+
+    @media only screen and (min-width: 620px) {
+        display: flex;
+        flex-wrap: wrap;
+        height: 250px;
+        width: auto;
+    }
+
+    @media only screen and (min-width: 911px) {
+        bottom: 0;
+        height: 130px;
+        justify-content: space-between;
+        width: auto;
+    }
+
+    @media screen and (min-width: 1000px) {
+        position: fixed;
+        width: 100%;
+        z-index: 4;
+    }
+}
+
+.footer {
+    &--title {
+        @include margin(30px, 25px, 11px, 25px);
+    }
+}
+
+.icon {
+    display: flex;
+
+    &--facebook {
+        height: 100%;
+        @include margin(0, 25px, 0, 5px);
+
+        &:hover {
```

Comment:
Since the hover effect here is consistent, you could apply it to the `icon--link` class that all the icons share?